build_tools/benchmarks/common/benchmark_definition.py

```diff
@@ -390,22 +411,79 @@ def from_json_object(json_object: Dict[str, Any]):
 
 
 @dataclass
-class BenchmarkRun(object):
-  """An object describing a single run of the benchmark binary.
+class BenchmarkLatency:
+  """Stores latency statistics for a benchmark run."""
+  mean: int
+  median: int
+  stddev: int
+  unit: str
 
-  - benchmark_info: a BenchmarkInfo object describing the benchmark setup.
-  - context: the benchmark context returned by the benchmarking framework.
-  - results: the benchmark results returned by the benchmarking framework.
+  def to_json_object(self) -> Dict[str, Any]:
+    return {
+        "mean": self.mean,
+        "median": self.median,
+        "stddev": self.stddev,
+        "unit": self.unit,
+    }
+
+  @staticmethod
+  def from_json_object(json_object: Dict[str, Any]):
+    return BenchmarkLatency(
+        json_object["mean"],
+        json_object["median"],
+        json_object["stddev"],
+        json_object["unit"],
+    )
+
+
+@dataclass
+class BenchmarkMetrics(object):
+  """An object describing the results from a single benchmark.
+
+  - context: additional JSON-compatible context and raw results returned by the
+      benchmarking framework.
+  - real_time: the real time latency statistics returned by the benchmarking
+      framework.
+  - cpu_time: the cpu time latency statistics returned by the benchmarking
+      framework.
   """
-  benchmark_info: BenchmarkInfo
   context: Dict[str, Any]
-  results: Sequence[Dict[str, Any]]
+  real_time: BenchmarkLatency
+  cpu_time: BenchmarkLatency
 
   def to_json_object(self) -> Dict[str, Any]:
     return {
-        "benchmark_info": self.benchmark_info.to_json_object(),
         "context": self.context,
-        "results": self.results,
+        "real_time": self.real_time.to_json_object(),
+        "cpu_time": self.cpu_time.to_json_object(),
+    }
+
+  def to_json_str(self) -> str:
```

Comment:
Do we need this method?

Reply:
It was carried over from `BenchmarkRun`. I removed all unused instances of `to_json_str`.